enhancements/etcd/cluster-etcd-operator.md

```diff
@@ -325,6 +345,42 @@ The cluster can function without intervention, but to fully restore 4.3, manual
     3. wait for the etcd-member to rejoin
     4. move to the next master.
 
+#### Migrating to IPs
+
+Because reasons.
+
+##### Etcd Clients
+
+Etcd clients such as KAS-o & OAS-o will compile a list of etcd grpc endpoint urls:
+
+1. From IPs found in `oc -n openshift-etcd get endpoints/host-etcd-2`:
+   * bootstrap member IP from `alpha.installer.openshift.io/etcd-bootstrap` annotation if available.
+   * cluster member IPs from **.spec.subsets[0].addresses[*n*].ip**.
+2. Local host urls: `https://120.0.0.1:2379` and `https://[::1]:2379`
+
+##### Etcd Serving Certificates
+
+Etcd client serving certificates will need to additionally be valid for `127.0.0.1` and `::1`.
+
+##### Etcd Peer URLs
+
+In release 4.4 etcd **peerUrls** will contain both IP and hostname variations in order to facilitate a downgrade.
+
+##### Handling Node IP Changes
+
+If a node's IP address changes, the etcd cluster will become non-functional. We recover by recovering a single etcd 
```

Comment:
break this apart into the two cases: 

1. all IPs change at the same time
2. one IP changes.